data-imports/import-aep.py

```diff
@@ -33,10 +33,10 @@ def main():
 
 def import_csv(args):
     log.info('Importing pre-created AEP csv...')
-    data_file = 'data/AEP_LIST.csv'
+    data_file = '/root/data/AEP_LIST.csv'
```

Comment:
Adding '/root' means it can't be run outside of Docker. We could just decide these scripts can't be run outside of docker, but I'd like to keep the option available. What's the working directory the script is running from in Docker, and how can we map things so the script can be run from the root of the repo?

(Either way the import scripts could use a README, I can work on that)